openpaas-james/apps/memory/pom.xml

```diff
@@ -96,13 +140,14 @@
                             <port>25</port> <!-- SMTP -->
                             <port>465</port> <!-- SMTP + STARTTLS -->
                             <port>587</port> <!-- SMTPS -->
-                            <port>4000</port> <!-- GLOWROOT, if activated -->
+                            <port>4000</port> <!-- GLOWROOT -->
                             <port>8000</port> <!-- WEBADMIN -->
                         </ports>
                         <appRoot>/root</appRoot>
                         <jvmFlags>
                             <jvmFlag>-Dlogback.configurationFile=/root/conf/logback.xml</jvmFlag>
                             <jvmFlag>-Dworking.directory=/root/</jvmFlag>
+                            <jvmFlag>-javaagent:/root/glowroot/glowroot.jar</jvmFlag>
```

Comment:
I don't get how it is disabled by default with this in the JIB declaration?

Reply:
See the wall diff @Arsnael :-)

Reply:
I messed up and did not remove the line in the end (I mean: did not commit it)